parsing/dialects/dialect1/src/test/java/org/apache/calcite/test/Dialect1ParserTest.java

```diff
@@ -1746,40 +1746,40 @@ private void checkExNotNull(SqlParserUtil.StringAndPos sap, Throwable thrown) {
     sql(sql).ok(expected);
   }
 
-  @Test void testInlineFormatSimpleIdentifier() {
+  @Test void testAlternativeCastAttributeSimpleIdentifier() {
```

Comment:
nit: Should we have the word "format" somewhere in this test name? (Here and other functions below).

Reply:
Sure I'll update them with Format